testsuite/tests/lib-format/print_array.ml

```diff
@@ -0,0 +1,33 @@
+(* TEST
+   include testing
+*)
+
+(*
+
+A test file for the Format module.
+
+*)
+
+open Testing;;
+open Format;;
+
+let say s = Printf.printf s;;
+
+let pp_print_intarray = pp_print_array ~pp_sep:(fun fmt () -> pp_print_char fmt ' ') pp_print_int;;
+
+try
+
+  say "empty\n%!";
+  test (asprintf "%a%!" pp_print_intarray [||] = "");
+
+  say "\nmisc\n%!";
+  test (asprintf "%a" pp_print_intarray [| 0 |]       = "0");
+  test (asprintf "%a" pp_print_intarray [| 0; 1; 2 |] = "0 1 2");
+  test (asprintf "%a" pp_print_intarray [| 0; 0 |]    = "0 0");
+
+  say "\nend of tests\n%!";
+
+with e ->
+  say "unexpected exception: %s\n%!" (Printexc.to_string e);
+  test false;
```

Comment:
It's a nit, but this isn't necessary - if the test fails, the exception would be displayed anyway